crates/turborepo-lib/src/task_graph/visitor/command.rs

```diff
@@ -269,11 +266,31 @@ impl<'a> CommandProvider for MicroFrontendProxyProvider<'a> {
     }
 }
 
+/// A trait for fetching package information required to execute commands
+pub trait PackageInfoProvider {
```

Comment:
This trait allows us to test the microfrontends command generation without building an entire package graph.